### ISSUE TITLE
Using SPDX UNLICENSED is not the same as UNLICENSE

### ISSUE THREAD

**Issue by d1onys1us** [privacy-scaling-explorations/zkevm-chain#96]
I noticed quite a few files using SPDX UNLICENSED at the top.

I just want to clear up if this is intentional, or if this is referring to https://unlicense.org, because they are technically very different and I am not sure if I am allowed to use this work. (lack of license or usage of UNLICENSED implies I do not have permission). 

sources:
- https://opensource.stackexchange.com/a/12412
- https://choosealicense.com/no-permission/
- https://github.com/spdx/spdx-spec/issues/49

Thanks!

**Comment by pinkiebell** [comment]
Yes, I intended to mean the unlicense.org. However, it is being used as a placeholder because we didn't decided yet which license we are going to use here. But it looks like it will be MIT. Consider this to be resolved soon 👌

**Comment by d1onys1us** [comment]
thanks @pinkiebell , this is definitely sufficient to be safer to use. FYI, as I am using this library, I am not sure who to attribute to. A root LICENSE.md would help if attribution is something desired here.

**Comment by pinkiebell** [comment]
If you want to include it, you can refer to [this](https://github.com/privacy-scaling-explorations) organization. With a link or just the name. Whatever you see fit 👌
